neurodamus/node.py

```diff
@@ -258,7 +258,7 @@ def read_population_offsets(cls, read_virtual_pop=False):
         return pop_offsets, alias_pop
 
     @classmethod
-    def _pop_offset_file(self, create=False):
+    def _pop_offset_file(cls, create=False):
```

Comment:
knitpick: since you changed `self` to `cls` without adding other changes in the class I suspect that this classmethod may be a staticmethod instead